packages/web3/test/e2e/sepolia/get_past_logs.test.ts

```diff
@@ -67,8 +67,8 @@ describe(`${getSystemTestBackend()} tests - getPastLogs`, () => {
 		const result = (
 			await web3.eth.getPastLogs(
 				{
-					fromBlock: 'earliest',
-					toBlock: 'latest',
+					fromBlock: 3229301,
```

Comment:
why these are changed?

Reply:
in this test, we try to check that it returns correct types.
from `earliest` to `latest` will always return an error. It's a very old block. but if we specify from to as a real block numbers it will work